Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1566,8 +1566,8 @@ std::unique_ptr<RenderStyle> TreeResolver::PositionOptions::currentOption() cons
     ASSERT(optionStyles[index].style);
 
     auto newStyle = RenderStyle::clonePtr(*optionStyles[index].style);
-    if (optionStyles[index].fallbackIndex)
-        newStyle->setLastSuccessfulPositionTryFallbackIndex(*optionStyles[index].fallbackIndex);
+    auto fallbackIndex = optionStyles[index].fallbackIndex;
+    newStyle->setLastSuccessfulPositionTryFallbackIndex(WTFMove(fallbackIndex));
```

Comment:
It is a bit odd factoring/naming that a function called currentOption() (which sounds like a getter) also sets things.

Reply:
This patch https://github.com/WebKit/WebKit/pull/50010 will separate the "used position option index" and "last successful position option index", so in that patch we'll set the "used position option index" at option generation time, so `::currentOption()` won't set it anymore